packages/code-du-travail-api/src/server/routes/items/index.js

```diff
@@ -54,27 +74,6 @@ router.get("/items/:source/:slug", async (ctx) => {
   };
 });
 
-/**
- * Return document matching the given id.
- *
- * @example
- * http://localhost:1337/api/v1/items/:id
- *
- * @param {string} :id The item id.
- * @returns {Object} Result.
- */
-router.get("/items/:id", async (ctx) => {
```

Comment:
you should refactor to keep this method as it is exactly what you try do do upper.